### PR TITLE
fix ggplot2 outputs

### PR DESCRIPTION
**Pull request by lecorguille**
A little confusion about outputs filename

heatmap2.xml
```xml
<command>
mv output_plot.pdf $output1
</command>
<configfiles>
    <configfile name="script">
ggsave(file='output_plot.pdf', width=gg_width, height=gg_height)
    </configfile>
</configfiles>
<outputs>
    <data name="output1" format="pdf" from_work_dir="Rplot.pdf"/>
</outputs>
```

### CODE REVIEW

**Comment by nsoranzo** [comment]
This is going to conflict with https://github.com/galaxyproject/tools-iuc/pull/1426 , can you check if that would solve the issue?

**Comment by bgruening** [comment]
I fixed in the meantime the conda issue.

File: tools/ggplot2/heatmap2.xml [hunk 1]
```diff
@@ -64,8 +63,7 @@ hclustfun=hclust_fun
   clabs = FALSE
 #end if
 
-pdf(file='output_plot.pdf')
-colorscale
+pdf(file='$output1')
```
Comment (bgruening) [inline tools/ggplot2/heatmap2.xml]: I think we need to have here the pdf file ending.

**Comment by bernt-matthias** [comment]
Apparently this has been solved elsewhere

https://github.com/galaxyproject/tools-iuc/blob/43f3e91f557114060e6efc69e1d4ea711bb1ea96/tools/ggplot2/macros.xml#L280
https://github.com/galaxyproject/tools-iuc/blob/43f3e91f557114060e6efc69e1d4ea711bb1ea96/tools/ggplot2/macros.xml#L79

**Pull request closed without merging**